docs/emoji-key.md

```diff
@@ -41,5 +41,5 @@ Emoji/Type | Represents | Comments
 📹 <br /> `video` | Videos | links to the video
 
 ## What's next
-- [Notes for repository maintainers](repository-maintainers)
-- [Automating the process using tools](tooling)
+- [Notes for repository maintainers](https://github.com/all-contributors/all-contributors/blob/master/docs/repository-maintainers.md)
+- [Automating the process using tools](https://github.com/all-contributors/all-contributors/blob/master/docs/tooling.md)
```

Comment:
Absolute paths are unnecessary here and will inevitably break the Docusaurus build.

Reply:
I had completely forgotten about the docusaurus build, sorry. The links in the md were not working; is there no way we can make the links work without appending .md manually?

Reply:
I don't think so.
@jakebolam @gr2m @sinchang Any ideas?